invalid-pr

M-ZubairAhmed opened the pull request:
#### Summary
- [x] Separated EmojiPickerSearch into separate components
- [x] Separated EmojiPickerCategories into separate components
- [x] Broken apart emojiCurrentResults into separate components

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-40296

#### Related Pull Requests
None

#### Screenshots
<!--
If the PR includes UI changes, include screenshots/GIFs.

For an easier comparison of UI changes a table (template below) can be used.

|  before  |  after  |
|----|----|
| <insert before screenshot here> | <insert after screenshot here> |

-->

#### Release Note
<!--
Add a release note for each of the following conditions:

* Config changes (additions, deletions, updates)
* API additions—new endpoint, new response fields, or newly accepted request parameters
* Database changes (any)
* Websocket additions or changes
* Anything noteworthy to a Mattermost instance administrator (err on the side of over-communicating)
* New features and improvements, including behavioural changes, UI changes and CLI changes
* Bug fixes and fixes of previous known issues
* Deprecation warnings, breaking changes, or compatibility notes

If no release notes are required write NONE. Use past-tense. Newlines are stripped.

Examples:

```
Added new API endpoints POST /api/v4/foo, GET api/v4/foo, and GET api/v4/foo/:foo_id.
```

```
Added a new config setting ServiceSettings.FooBar. Added a new column Foo to the Users table.
```

```
NONE
```
-->
```release-note
NONE
```

esethna commented:
Hey @M-ZubairAhmed, thanks for working on this! Adding myself as a reviewer as I'd like to chat through the optimizations we're making in the new year. Not urgent before the holidays

M-ZubairAhmed commented:
Hey thanks for adding yourself, going forward as @jwilander  suggested I would split this into multiple small prs to master to facilitate easy review will add you as reviewer to all of them too

M-ZubairAhmed committed:
init

M-ZubairAhmed committed:
no tests

M-ZubairAhmed committed:
slight index improvment

M-ZubairAhmed committed:
merge master

M-ZubairAhmed committed:
remove async

tests

in prog

file renmae

category section

minor diff

custom button

rev

eslint

block resetcursor when not needed

common funcs

virt step 1

virtualize var

stash

neallred committed:
fix Dm name in file preview modal info bar (#9547)

use state.show & top level onExited for invitation modal (#9550)

[MM-40407] - Do not show the Renew Now if the license id does not exist in the portal (#9544)

* [MM-40407] - Do not show the Renew Now if the license id does not exist in the portal

* fix translation strings

* add unit test

* improvements

MM-40600 - use custom type and set correct value to api response (#9546)

Automatic Merge

MM-40218 - start trial from team edition, do upgrade and request license (#9454)

* MM-40218 - start trial from team edition, do upgrade and request license

* add i18n texts

* fix snapshots

* show restarting text and define which button was clicked

* prevent the preference to get saved

* hide the update and request trial banner if normal button is clicked

* fix tests and snapshots

MM-40114/MM-40545 Clean up MobileSidebarHeader and have it use getIsMobileView (#9512)

* MM-40114 Clean up LegacySidebarHeader and remove usage of Utils.isMobile

* MM-40114 Rename LegacySidebarHeader to MobileSidebarHeader and remove unused dropdown

* MM-40545 Fix Account Settings shortcut in desktop view

* Apply suggestions from code review

Co-authored-by: Jelena Gilliam <52937121+jgilliam17@users.noreply.github.com>
Co-Authored-By: Pablo Velez Vidal <pablo.velez@mattermost.com>

M-ZubairAhmed commented:
/update-branch
